ui/page/privacy/account_mixer_page.go

```diff
@@ -68,175 +68,176 @@ func (pg *AccountMixerPage) OnNavigatedTo() {
-	pg.allowUnspendUnmixedAcct.SetChecked(isSpendUnmixedFunds)
+	/*
+		isSpendUnmixedFunds := pg.wallet.ReadBoolConfigValueForKey(load.SpendUnmixedFundsKey, false)
+		pg.allowUnspendUnmixedAcct.SetChecked(isSpendUnmixedFunds)
+	*/
 }
 
-// Layout draws the page UI components into the provided layout context
-// to be eventually drawn on screen.
-// Part of the load.Page interface.
-func (pg *AccountMixerPage) Layout(gtx layout.Context) layout.Dimensions {
-	d := func(gtx C) D {
-		sp := components.SubPage{
-			Load:       pg.Load,
-			Title:      values.String(values.StrStakeShuffle),
-			WalletName: pg.WL.SelectedWallet.Wallet.Name,
-			BackButton: pg.backButton,
-			InfoButton: pg.infoButton,
-			Back: func() {
-				pg.ParentNavigator().CloseCurrentPage()
-			},
-			InfoTemplate: modal.PrivacyInfoTemplate,
-			Body: func(gtx layout.Context) layout.Dimensions {
-				widgets := []func(gtx C) D{
-					func(gtx C) D {
-						return components.MixerInfoLayout(gtx, pg.Load, pg.WL.SelectedWallet.Wallet.IsAccountMixerActive(),
-							pg.toggleMixer.Layout, func(gtx C) D {
-								mixedBalance := "0.00"
-								unmixedBalance := "0.00"
-								accounts, _ := pg.WL.SelectedWallet.Wallet.GetAccountsRaw()
-								for _, acct := range accounts.Acc {
-									if acct.Number == pg.WL.SelectedWallet.Wallet.MixedAccountNumber() {
-										mixedBalance = dcrutil.Amount(acct.TotalBalance).String()
-									} else if acct.Number == pg.WL.SelectedWallet.Wallet.UnmixedAccountNumber() {
-										unmixedBalance = dcrutil.Amount(acct.TotalBalance).String()
-									}
-								}
-
-								return components.MixerInfoContentWrapper(gtx, pg.Load, func(gtx C) D {
-									return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-										layout.Rigid(func(gtx C) D {
-											return layout.Flex{Spacing: layout.SpaceBetween, Alignment: layout.Middle}.Layout(gtx,
-												layout.Rigid(func(gtx C) D {
-													txt := pg.Theme.Label(values.TextSize14, "Unmixed balance")
-													txt.Color = pg.Theme.Color.GrayText2
-													return txt.Layout(gtx)
-												}),
-												layout.Rigid(func(gtx C) D {
-													return components.LayoutBalance(gtx, pg.Load, unmixedBalance)
-												}),
-											)
-										}),
-										layout.Rigid(func(gtx C) D {
-											return layout.Center.Layout(gtx, pg.Theme.Icons.ArrowDownIcon.Layout24dp)
-										}),
-										layout.Rigid(func(gtx C) D {
-											return layout.Flex{Spacing: layout.SpaceBetween, Alignment: layout.Middle}.Layout(gtx,
-												layout.Rigid(func(gtx C) D {
-													t := pg.Theme.Label(values.TextSize14, "Mixed balance")
-													t.Color = pg.Theme.Color.GrayText2
-													return t.Layout(gtx)
-												}),
+func (pg *AccountMixerPage) MixerInfoLayout(gtx C, l *load.Load, mixerActive bool, button layout.Widget) D {
+	mixedBalance := "0.00"
```

Comment:
Just 0, Need to agree on how to display the balance

Reply:
As I said earlier, this PR focuses on the UI and not functionality